functions/Move-DbaDbFile.ps1

```diff
@@ -0,0 +1,438 @@
+                                        Copy-Item -Path $physicalName -Destination $destination -ErrorAction Stop
+
+                                    } catch {
+                                        $failed = $true
+
+                                        Write-Message -Level Important -Message "ERROR: Could not copy file. $_"
+                                    }
+                                }
+                            }
+                        } else {
+                            # Use Remoting PS to run the command on the server
+                            try {
+                                if ($PSCmdlet.ShouldProcess($database, "Copying file $physicalName to $destination using remote PS on $ComputerName")) {
+                                    $scriptBlock = {
+                                        $physicalName = $args[0]
+                                        $destination = $args[1]
+
+                                        # Version 1 will yield - "The remote use of BITS is not supported." when using Remoting PS
+                                        if ((Get-Command -Name Start-BitsTransfer).Version.Major -gt 1) {
+                                            Write-Verbose "Try copying using Start-BitsTransfer."
+                                            Start-BitsTransfer -Source $physicalName -Destination $destination -ErrorAction Stop
+                                        } else {
+                                            Write-Verbose "Can't use Bits. Using Copy-Item instead"
+                                            Copy-Item -Path $physicalName -Destination $destination -ErrorAction Stop
+                                        }
+
+                                        Get-Acl -Path $physicalName | Set-Acl $destination
+                                    }
+                                    Invoke-Command2 -ComputerName $ComputerName -Credential $SqlCredential -ScriptBlock $scriptBlock -ArgumentList $physicalName, $destination
+                                }
+                            } catch {
+                                # Try using UNC paths
+                                try {
+                                    $physicalNameUNC = Join-AdminUnc -ServerName $ComputerName -Filepath $physicalName
+                                    $destinationUNC = Join-AdminUnc -ServerName $ComputerName -Filepath $destination
+
+                                    if ($PSCmdlet.ShouldProcess($database, "Copying file $physicalNameUNC to $destinationUNC using UNC path for $ComputerName")) {
+
+                                        try {
+                                            Write-Message -Level Verbose -Message "Try copying using Start-BitsTransfer with UNC paths."
+                                            Start-BitsTransfer -Source $physicalNameUNC -Destination $destinationUNC -ErrorAction Stop
```

Comment:
Should likely allow for `$Credential` as well but could leave it alone until we start seeing the use of the command for this use case.